sync_diff_inspector/diff.go

```diff
@@ -100,6 +113,13 @@ func (df *Diff) init(cfg *Config) (err error) {
 
 // CreateDBConn creates db connections for source and target.
 func (df *Diff) CreateDBConn(cfg *Config) (err error) {
+	if len(df.subTaskCfgs) != 0 {
```

Comment:
how about move this if block into `if len(cfg.DMAddr) != 0 {}` in `(*Diff).init`? so `CreateDBConn` won't do prepare jobs.

Reply:
update in fbe1068